.zuul.yaml

```diff
@@ -0,0 +1,22 @@
+- job:
+    name: dnf-run-tests
+    description: Run DNF tests
+    roles:
+      - zuul: zuul-distro-jobs
+    requires: repo
+#    require-projects:
+#      - rpm-software-management/ci-dnf-stack
+    branches:
+      - master
+    vars:
+      dnf_master_copr: fbo/dnf-master
+    run: zuul-ci/run-tests.yaml
+    nodeset: fedora-31-vm
+
+- project:
```

Comment:
This is the project's pipeline config. Where we tell Zuul, in the "check" pipeline, run the dnf-rpm-build and dnf-run-tests jobs. The dnf-run-tests is a child job because we define a dependency on dnf-rpm-build. dnf-run-tests job will start after dnf-rpm-build is done.